### PR TITLE
First prototype of 'generate_all_protocols.py' script

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

First prototype of 'generate_all_protocols.py' script

## Fixes

n/a

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments
n/a

### CODE REVIEW

File: aea/protocols/signing/protocol.yaml [hunk 1]
```diff
@@ -10,10 +10,10 @@ fingerprint:
   __init__.py: QmcCL3TTdvd8wxYKzf2d3cgKEtY9RzLjPCn4hex4wmb6h6
   custom_types.py: Qmc7sAyCQbAaVs5dZf9hFkTrB2BG8VAioWzbyKBAybrQ1J
   dialogues.py: QmQ1WKs3Dn15oDSwpc4N8hdADLxrn76U4X5SiLAmyGiPPY
-  message.py: QmRLWAYfjCQmdg2hH8R5R63DKYaDrzuX4dVTFqNHuyjawq
-  serialization.py: QmZrztNBaWA6B5wJHQWfM2g6opPtvsEXtqytzmxjKWm7Sb
-  signing.proto: QmcxyLzqhTE9xstAEzCVH17osbLxmSdALx9njmuPjhjrvZ
-  signing_pb2.py: QmY3Ak5ih5zGvKjeZ5EnzrGX4tMYn5dWpjPArQwFeJpVKu
+  message.py: QmWMUnBpZxBq1PDZ6JsR9vwaKdywBHvL5hghekWeph8j9B
```
Comment (marcofavorito) [inline aea/protocols/signing/protocol.yaml:13]: `message.py` changed only because of the order of import statements

Comment (marcofavorito) [reply aea/protocols/signing/protocol.yaml:13]: (by the way, this happens only in the default protocols)

**Comment by marcofavorito** [comment]
Fixes #1721

File: aea/protocols/signing/signing.proto [hunk 2]
```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 
-package fetch.aea.Signing;
+package fetchai.aea.fetchai.signing;
```
Comment (DavidMinarsch) [inline aea/protocols/signing/signing.proto:3]: Why the fetchai twice? Why not simply `aea.author_name.package_name`? I don't think aea should be tied to fetchai like this

**Review by DavidMinarsch — APPROVED** [review]
LGTM, apart from one comment

File: aea/protocols/signing/signing.proto (hunk 2 above)
Comment (marcofavorito) [reply aea/protocols/signing/signing.proto:3]: ok, I'm going to remove the first `fetchai`

**Pull request merged**